modules/oadp-setting-resource-limits-and-requests.adoc

```diff
@@ -38,4 +38,9 @@ spec:
             memory: 256Mi
 ----
 <1> Specify the node selector to be supplied to Velero podSpec.
-<2> The `resourceAllocations` listed are for average usage.
\ No newline at end of file
+<2> The `resourceAllocations` listed are for average usage.
+
+[NOTE]
+====
+In OADP version 1.3 and later, Kopia can be used as an uploader. Kopia can be resource intensive, and you might need to adjust the CPU and memory requirements accordingly.
```

Comment:
s/In OADP version 1.3 and later, Kopia can be used as an uploader/Kopia is now an option in OADP 1.3. and later releases.  Kopia is used for both file system backups and data mover use cases.   Kopia is more resource intensive than Restic, and you might need to adjust the CPU and memory requirements accordingly.

Reply:
maybe clarify that with the  (native) data mover, you do not have any other option than Kopia.
@weshayutin

Reply:
Yes.. I agree. Mentioning that kopia is the only option for data mover use cases is important.  @CarmiWisemon do you have enough info to slightly rewrite?

Reply:
@weshayutin @amastbau 
I made the update you requested.